src/controllers/libraryController.ts

```diff
@@ -55,11 +55,13 @@ export const createLibrary = async (req: Request, res: Response) => {
 /**
  * Adds a game to a specified library
  *
+ * TODO: game should be added to "master library" in addition to specific library here.
  */
 export const addToLibrary = async (req: Request, res: Response) => {
   try {
     const libraryId = req?.params?.libraryId;
     const gameId = req.body.gameId;
+    // todo: create game instance
```

Comment:
Is this a separate ticket?

Reply:
ye - https://gaming-backlog.atlassian.net/jira/software/projects/GB/boards/1?selectedIssue=GB-28